deploy/runtools/firesim_topology_with_passes.py

```diff
@@ -521,13 +521,18 @@ def run_workload_passes(self, use_mock_instances_for_testing: bool) -> None:
         self.boot_simulation_passes(False, skip_instance_binding=True)
 
         @parallel
-        def monitor_jobs_wrapper(run_farm, completed_jobs: List[str], teardown: bool, terminateoncompletion: bool, job_results_dir: str) -> Dict[str, Dict[str, bool]]:
+        def monitor_jobs_wrapper(
```

Comment:
I can move this back to the old version... wasn't sure if we care about line length limits.